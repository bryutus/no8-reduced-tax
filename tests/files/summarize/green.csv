コード,得意先名,商品コード,商品名,テキスト30,数量,,売価金額,,上代金額,,txtRunningSum
0,得意先A,,,,,,,,,,
,,111,AFC会費,,1.00,,0,,"4,500",,1
,,577,MLFダークカラー,,1.00,,0,,"1,500",,2
,,93,ジオリナ ビエッセ レギュラー,,1.00,,0,,"1,000",,3
,,85,ジオリナ 酵素,,7.00,,0,,"¥70,000",,4
,,1120,アルソアクィーンシルバー,,1.00,,0,,"¥4,000",,5
,,156,セット3 I,,2.00,,0,,"¥22,000",,6
,,89,ジオリナ　霊芝　（ラージ）,,-1.00,,"(18,000)",,"(30,000)",,7
,,,,,テキスト25:,13.00,テキスト26:,\0,テキスト33:,"\73,000",
8,得意先B,,,,,,,,,,
,,,,,0.00,,0 ,,0 ,,1
,,1121,ローションII,,1.00,,0,,"¥3,000",,1
,,932,リップステイクオレンジ,,1.00,,0,,"¥1,000",,2
,,72,ライフォリジン Fe,,1.00,,0,,"¥3,500",,3
,,157,セット3 II,,2.00,,0,,"¥22,200",,4
,,1130,クレイパック,,4.00,,0,,"¥14,000",,5
,,917,ベスト4ローションI,,3.00,,0,,"¥15,000",,6
,,918,ベスト4ローションII,,-1.00,,(0),,"(5,000)",,7
,,919,ベスト4セルローション,,4.00,,0,,"¥20,000",,8
,,,,,テキスト25:,16.00,テキスト26:,\0,テキスト33:,"\73,700",
,,,,,テキスト15:,29.00,テキスト16:,\0,テキスト34:,"\151,700",
