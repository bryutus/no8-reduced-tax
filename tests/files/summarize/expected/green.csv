得意先名,化粧品,健食,合計,酵素,石鹸,パック,ローション,ビッグローション,エッセンス,セット3,ベスト4
得意先A,"27,500","41,000","68,500",7,1,,,,,2,
得意先B,"70,200","3,500","73,700",,,4,1,6,,2,
合計,"97,700","44,500","142,200",7,1,4,1,6,,4,
